### PR TITLE
Add OSX to contacts and calendar apps

### PR DESCRIPTION
**Pull request by Thatoo**
Picture os steps are missing because I don't know how to upload them. I send them to Frank.

### CODE REVIEW

File: src/documents/en/mobile/calendar.html.md [hunk 1]
```diff
@@ -138,6 +138,33 @@ Open the menu and click the button to add a new calendar.
 <br />
 That's it! You can seamlessly manage your events from your mobile and your Cozy.
 
+## OXS
+
+Inside Calendar app, click on Calendrier and click on "Add Account" :
```
Comment (frankrousseau) [inline src/documents/en/mobile/calendar.html.md]: Calendrier is not displayed in English version.

File: src/documents/en/mobile/calendar.html.md [hunk 2]
```diff
@@ -138,6 +138,33 @@ Open the menu and click the button to add a new calendar.
 <br />
 That's it! You can seamlessly manage your events from your mobile and your Cozy.
 
+## OXS
```
Comment (frankrousseau) [inline src/documents/en/mobile/calendar.html.md]: I think you mean OSX ?

File: src/documents/en/mobile/contacts.html.md [hunk 3]
```diff
@@ -117,6 +117,34 @@ Finally, fill the form with the credentials you got from Cozy's Sync app:
 Congratulations, your mobile device is now synchronized with your Cozy!
 <center>![iOS step 5](/assets/images/mobile/ios/contacts/step5.jpg)</center>
 
+## OSX
+
+#### Configure your CardDav account
+Once the software "Contacts" open, click on Contacts and "Ajouter un compte" :
+<center>![OSX step 1](/assets/images/mobile/osx/contacts/step1.jpg)</center>
+
+<br />
```
Comment (frankrousseau) [inline src/documents/en/mobile/contacts.html.md]: Please don't use <br /> with markdown. Two blank lines are enough.

**Comment by frankrousseau** [comment]
Thank you @Thatoo for your modification proposal! I made some comments on what you sent. It requires some fixes before we can publish it.

File: src/documents/en/mobile/calendar.html.md [hunk 4]
```diff
@@ -138,6 +138,33 @@ Open the menu and click the button to add a new calendar.
...
+<center>![OSX step 1](/assets/images/mobile/oxs/calendar/step1.jpg)</center>
+
+<br />
+<br />
+Then select "Add CalDav Account", in the Calendars section :
+<center>![OSX step 2](/assets/images/mobile/oxs/calendar/step2.jpg)</center>
+
+<br />
+<br />
+Select the Type of contact : Manual.
```
Comment (frankrousseau) [inline src/documents/en/mobile/calendar.html.md]: In english `:` are not prefixed with a space.

File: src/documents/en/mobile/contacts.html.md (hunk 3 above)
Comment (Thatoo) [inline src/documents/en/mobile/contacts.html.md]: -----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

je ne comprends pas ce que tu veux me dire cette fois.
Merci pour les autres corrections, désolé, je n'aurais pas dû le faire
si tard hier pour l'anglais.

Nathanaël Leprette

Le 04/06/2015 13:05, Frank Rousseau a écrit :
> In src/documents/en/mobile/contacts.html.md 
> <https://github.com/cozy/cozy-docs/pull/133#discussion_r31707769>:
> 
>> @@ -117,6 +117,34 @@ Finally, fill the form with the credentials
>> you got from Cozy's Sync app: Congratulations, your mobile device
>> is now synchronized with your Cozy! <center>iOS step
>> 5</center>
>> 
>> +## OSX + +#### Configure your CardDav account +Once the software
>> "Contacts" open, click on Contacts and "Ajouter un compte" : 
>> +<center>OSX step
>> 1</center> + +<br
>> />
> 
> Please don't use with markdown. Two blank lines are enough.
> 
> — Reply to this email directly or view it on GitHub 
> <https://github.com/cozy/cozy-docs/pull/133/files#r31707769>.
> 
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2.0.22 (GNU/Linux)

iQEcBAEBCAAGBQJVcCrZAAoJEEunEQsIr90BAV0H+wdKt4r3EpzSU7KJ9shK6fyW
p6NVG1RAghtNgzBPHaWrHoC9MvKoTvnj/R21s+9ZtugNUlLpTRm9pY4nFbVxQR2w
eUNP7WRfytUmA/KG3zV8zBFeRvMDwADIWMIWNT3fZkBTWE/B5HkdDh/GnCjkLLHO
lHvDEzeYPTLQ1rmtAP3wvZojBbjdItHQh+yrErznQeeM2AqN8YYKpRmNBexFKPXI
S4oNpIOpi1dAwe5hMz3b5rdy/3R3oB6YjsJaVVIpja3FA6G4n4Y8h+ZavKf/G/Dg
TYsZMHCjm9FRSM1Ym8FUf8wIt9jRU79tzSZI0HQWLPZ/crd8fmuBT1/b4jr8L2o=
=de+9
-----END PGP SIGNATURE-----

**Comment by frankrousseau** [comment]
Awesome! Thank you a lot @Thatoo

**Pull request merged**